src/language/typescript/languageFeatures.ts

```diff
@@ -474,7 +474,7 @@ export class SuggestAdapter extends Adapter implements languages.CompletionItemP
 			}
 
 			const tags: languages.CompletionItemTag[] = [];
-			if (entry.kindModifiers?.indexOf('deprecated') !== -1) {
+			if (entry.kindModifiers && entry.kindModifiers.indexOf('deprecated') >= 0) {
```

Comment:
```suggestion
			if (entry.kindModifiers && entry.kindModifiers.indexOf('deprecated') !== -1) {
```

`indexOf(...) !== -1` is a common pattern.